c7n/filters/core.py

```diff
@@ -432,8 +432,7 @@ def process_value_type(self, sentinel, value):
                 # EMR not having more functionality.
                 try:
                     value = parse(value, default=datetime.now(tz=tzutc()))
-
-                except (AttributeError, TypeError):
+                except:
```

Comment:
i'd rather just add the ValueError to the list, afaics between the three that should cover. unconditional exceptions can mask other things. at the very least on a blank it needs an inline comment.